Cargo.toml

```diff
@@ -55,6 +55,7 @@ mint = { version = "0.5.6", optional = true }
 [dev-dependencies]
 image = { version = "0.24.0", default-features = false, features = ["png"] }
 simple_logger = { version = "2.1.0", default_features = false }
+web-sys = { version = "0.3.22", features = ['CanvasRenderingContext2d'] }
```

Comment:
Can you put this in a `[target.'cfg(target_arch = "wasm32")'.dev-dependencies]` instead?